platform/ui/src/components/InputGroup/InputGroup.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import {
+  InputText,
+  InputDateRange,
+  InputMultiSelect,
+  InputLabelWrapper,
+} from '@ohif/ui';
+
+const InputGroup = ({
+  filtersMeta,
```

Comment:
Sharing some thoughts:

- About the `filter` prefix on `filtersValues`, `filtersMeta`... - Currently, we use `InputGroup` in the filter, but it is not necessarily tied to a filter. Maybe we can use a more generic name for these props.

- setFilterValues, setFilterSorting - I'm not sure if this is the best approach (passing the hook itself to the component). We can avoid passing it and avoid extra logic in the component if we pass something like a `onValueChange` and `onSortingChange`, and the application will be responsible for handling the logic when something changes.

What do you think? If you have any question, we can discuss offline.

Reply:
Both changes make sense, I will work in this.

Reply:
Code updated. 
Nice callout, thanks 👍